tests/verify_object_limits.erl

```diff
@@ -128,3 +132,49 @@ verify_sibling_limits(C, Node1) ->
     lager:info("Result when too many siblings : ~p", [Res]),
     ?assertMatch({error,_},  Res),
     ok.
+
+verify_readrepair_ignore_max_size(C, Node1) ->
+    % Add intercept to treat all vnode puts as readrepairs
+    Intercept = {riak_kv_vnode, [{{put, 6}, put_as_readrepair},{{coord_put,6}, coord_put_as_readrepair}]},
+    ok = rt_intercept:add(Node1, Intercept),
+    timer:sleep(100),
```

Comment:
@bsparrow435 is this timer:sleep call necessary? I don't think you're supposed to have to sleep after adding an intercept (or if we are, I would hope there's some way of doing it in a deterministic way, rather than just hoping 100 ms is long enough to wait). I tried taking this out - along with the other timer:sleep call you added down below on line 166 - and the test seems to run fine without them.